### PR TITLE
Dev guide

### PR DESCRIPTION
**Pull request by lxgr-linux**
Added DevGuide.md for further documentation.

### CODE REVIEW

File: DevGuide.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        "ico": [{  # A list of dictionaries containing a displayed string and a color, all those strings will be layered over each other and represent the Pokete in the fight
+            "txt": """ +-------+
+ | o   o |
+ |  www  |
+ +-------+ """,
+            "esc": None}],
+    },
+```
+
+For more examples on the dictionaries see `pokete_data/poketes.py`.
```
Comment (MaFeLP) [inline DevGuide.md:37]: Add a reference:
```suggestion
For more examples on the dictionaries see [`pokete_data/poketes.py`](./pokete_data/poketes).
```

File: DevGuide.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+### Types
+Only the first type in the `"types"` list is the Poketes actual displayed type, that determines the Poketes effectivity. The other types in the list are just secondary types that determine what generic attacks a Pokete is able to learn. Those can also be subtypes. The main type can't!
+For a list of all types and subtypes see `pokete_data/types.py`
```
Comment (MaFeLP) [inline DevGuide.md:41]:
```suggestion
For a list of all types and subtypes see [`pokete_data/types.py`](./pokete_data/types.py)
```

File: DevGuide.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+### Learning attacks
+In addition the attacks in the `"attacks"` and the `"pool"` lists, a Pokete can learn all generic attacks of the Poketes types.
+
+
+## Adding attacks
+Every attack has an entry the `attacks` dictionary in `pokete_data/attacks.py` every entry needs to have this structure:
```
Comment (MaFeLP) [inline DevGuide.md:48]:
```suggestion
Every attack has an entry the `attacks` dictionary in [`pokete_data/attacks.py `](./pokete_data/attacks.py) every entry needs to have this structure:
```

File: DevGuide.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+        "min_lvl": 0,  # The minimal level a Poketes has to have to learn the attack
+        "desc": "Makes the enemy weaker.",  # The attacks description
+        "types": ["poison"],  # The attacks types
+        "effect": "poison",  # The effect the enemy gets when the attack is used, default is None 
+        "is_generic": False,  # Whether or not the attack can be learned by any Pokete of its types
+        "ap": 10,  # The attack points the attack has, so the amount of times the attack can be used by a Pokete until the Pokete has to be healed
+    },
+```
+
+For more examples on the dictionaries see `pokete_data/attacks.py`.
```
Comment (MaFeLP) [inline DevGuide.md:67]:
```suggestion
For more examples on the dictionaries see [`pokete_data/attacks.py`](./pokete_data/attacks.py).
```

File: DevGuide.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+### Types and learning
+Only the first type in the `"types"` list is the attacks real type and determines its effectivity. The other type can only be subtypes. A generic attack can only be learned by a Poketes that has all its types. An ungeneric attack can only be learned if the attack is in the Poketes `"pool"` or `"attacks"` list.
+
+### Effects
+The effect given in the `"effect"` attribute has to be the `c_name` of an effect listed in `pokete_classes/types.py` or `None`.
```
Comment (MaFeLP) [inline DevGuide.md:73]:
```suggestion
The effect given in the `"effect"` attribute has to be the `c_name` of an effect listed in [`pokete_classes/types.py`](./pokete_classes/types.py ) or `None`.
```

**Review by MaFeLP — CHANGES_REQUESTED** [review]

**Comment by lxgr-linux** [comment]
> Should this also be added to the gh-page?

Yes

**Review by MaFeLP — APPROVED** [review]

**Pull request merged**